File: programs/containers/list.cpp
#include <stdio.h>
#include <unistd.h>
#include <time.h>
#include <stdlib.h>
#include <iostream>
#include <list>
#include <vector>

int main()
{
    srand(time(NULL));
    std::list<int> test;
    for (int i = 0; i < 100000000; ++i)
    {
        test.push_back(rand());
    }
    printf("Done\n");
    sleep(20);
    printf("Sorting start\n");
    test.sort();
    printf("Sorting Done\n");
    sleep(10);
}

File: programs/containers/list.py
import random
import time

def main():
    random.seed()
    test = [random.randint(0, 2**31 - 1) for _ in range(100000000)]
    print("Done")
    time.sleep(20)
    print("Sorting start")
    test.sort()
    print("Sorting Done")
    time.sleep(10)

if __name__ == "__main__":
    main()